MemberExpectations.md

```diff
@@ -0,0 +1,63 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the
+[Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
+but that supports the vision, mission and operating principles of the Node.js project.
```

Comment:
Is the `vision, mission and operating principles` something that is going to be written down?

Reply:
It would be a good idea, but I think in order to avoid delaying this too long they could come afterwards.